tests/console/config/config_test.php

```diff
@@ -14,16 +14,14 @@
 use Symfony\Component\Console\Application;
 use Symfony\Component\Console\Tester\CommandTester;
 
-class phpbb_console_command_config_test extends phpbb_test_case
+class config_test extends phpbb_test_case
```

Comment:
Class name shouldn't be changed. (we should probably move tests to namespaces at some point ;) )

Reply:
I changed it to be able to run tests individually without having to run all test suite. File names in tests are not consistent with the class names.

![](https://i.imgur.com/7XNETHM.png)